### PR TITLE
pg_repack 1.5.2: Allow to run pg_repack by non-superuser

### PR DESCRIPTION
**Pull request by za-arthur**
## What kind of change does this PR introduce?

- upgrade `pg_repack` from `1.5.0` to `1.5.2`.

## Additional context

New version will allow to run `pg_repack` by a non-superuser. Conversation on slack:
https://supabase.slack.com/archives/C02FHG9QQAF/p1730124043199329

PR on `pg_repack` repo:
https://github.com/reorg/pg_repack/pull/431

`1.5.2` doesn't have release or tag yet, therefore instead of the tag commit id is used (`85b64c6d4f599b2988343c4e7121acab505c9006`).

## Action Items

- [x] **New extension releases** were Checked for any breaking changes
- [x] **Extensions compatibility** Checked
    * Proceed to [extensions compatibility testing](#extensions-compatibility-testing), mark as done after everything is completed
- [ ] **Backup and Restore** Checked
    * Proceed to [backup testing](#backup-testing) while extensions are enabled
        - After every restore, re-run the tests specified at point [3.1](#extensions-compatibility-testing)

### Extensions compatibility testing

1. Enable every extension
    1. Check Postgres’ log output for any error messages while doing so
        1. This might unearth incompatibilities due to unsupported internal functions, missing libraries, or missing permissions
2. Disable every extension
    1. Check Postgres’ log output for any cleanup-related error messages
3. Re-enable each extension
    1. Run basic tests against the features they offer, e.g.:
        1. `pg_net` - execute HTTP requests
        2. `pg_graphql` - execute queries and mutations
        3. …to be filled in

### Backup Testing

Follow the testing steps steps for all the following cases:

- Pause on new Postgres version, restore on new Postgres version
- Pause on older Postgres version, restore on new Postgres version
- Run a single-file backup backup, restore the backup

#### Testing steps

1. Generate dummy data 
    * the ‘Countries’ or ‘Slack clone’ SQL editor snippets are decent datasets to work with, albeit limited
2. Save a db stats snapshot file
    * Do this by running `supa db-stats gather -p <project_ref>`
3. Backup the database, through pausing the project, or otherwise
4. Restore the backup, through unpausing the project or cli
5. Check the data has been recovered successfully
    1. Visual checks/navigating through the tables works
    2. Run `supa db-stats verify` against the project and the previously saved file

### CODE REVIEW

File: ansible/tasks/postgres-extensions/27-pg_repack.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
    git:
      repo: https://github.com/reorg/pg_repack.git
      dest: /tmp/pg_repack
-     version: "ver_{{ pg_repack_release }}"
+     version: "{{ pg_repack_commit }}"
```
Comment (steve-chavez) [inline ansible/tasks/postgres-extensions/27-pg_repack.yml]: Q: Is this ansible file still used for postgres extensions? I thought we only do the Nix change nowadays

**Comment by steve-chavez** [comment]
~~Would it be possible to add a test for ensuring this works?~~

~~There are some example tests on https://github.com/supabase/postgres/tree/develop/nix/tests/sql~~

Oh, I see there's already a test upstream. Nevermind.

**Review by steve-chavez — APPROVED** [review]
LGTM :+1:

File: ansible/tasks/postgres-extensions/27-pg_repack.yml (hunk 1 above)
Comment (za-arthur) [reply ansible/tasks/postgres-extensions/27-pg_repack.yml]: Thanks for the review. Indeed, I wasn't sure if it was necessary. I reverted that change. I also bumped postgres release, it seems it is necessary to do on bumping an extension version.

**Review by tomashley — APPROVED** [review]

**Pull request merged**